### PR TITLE
i18n(fr): create error code references part 1

### PR DESCRIPTION
**Pull request by cravend**
#### What kind of changes does this PR include?

- Translated content

#### Description

- Updated main `error-reference.mdx` page to have new error links
- Translated:
  - content collection errors
  - cli errors
  - css errors

To do:
- Astro errors
- Markdown errors: #3387

### CODE REVIEW

**Review by dreyfus92 — APPROVED** [review]
Bon travail @cravend, LGTM ! ✅

**Review by thomasbnt — APPROVED** [review]
LGTM ! Good !

File: src/content/docs/fr/reference/errors/invalid-content-entry-slug-error.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+---
+
+
+> `COLLECTION_NAME` → `ENTRY_ID` a un slug invalide. `slug` doit être un string. (E09002)
+
+## Qu'est-ce qui a mal tourné ?
+Une entrée dans src/content/ a un slug invalide. Ce champ est réservé pour générer des slugs d'entrée, et doit être un string lorsqu'il est présent.
+
+**Voir aussi :**
+-  [Le champ slug d'entrée réservé](/en/guides/content-collections/)
```
Comment (Yan-Thomas) [inline src/content/docs/fr/reference/errors/invalid-content-entry-slug-error.mdx:14]:
```suggestion
- [Le champ slug d'entrée réservé](/fr/guides/content-collections/)
```

**Review by Yan-Thomas — APPROVED** [review]
Thanks for another amazing PR @cravend! Also thanks for the review peeps, LGTM 🚀

**Pull request merged**